src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -147,7 +147,28 @@ export default function Inbox ({ history: { push } }) {
   const refundAllDeclinedTransactions = useRefundAllDeclinedTransactions()
   const [counterpartyNotFound, setCounterpartyNotFound] = useState(true)
   const [isDeclinedTransactionModalVisible, setIsDeclinedTransactionModalVisible] = useState(false)
-  const [modalTransaction, setModalTransaction] = useState(null)
+  const [confirmationModalProperties, setConfirmationModalProperties] = useState({
+    shouldDisplay: false,
+    transactions: [],
+    action: '',
+    hasConfirmed: false
```

Comment:
this should be stored in a const

`const defaultConfirmationModalProperties = { ... }`

and used here and in the definition of `resetDefaultModalTransaction`

Reply:
also, afaict we're not actually using `hasConfirmed` at all